be/src/olap/rowset/rowset_meta.h

```diff
@@ -276,6 +277,22 @@ class RowsetMeta {
         return _rowset_meta_pb.alpha_rowset_extra_meta_pb();
     }
 
+    void set_tablet_schema(const TabletSchema* tablet_schema) {
+        TabletSchemaPB* ts_pb = _rowset_meta_pb.mutable_tablet_schema();
+        tablet_schema->to_schema_pb(ts_pb);
+        CHECK(_schema == nullptr);
+        _schema = std::make_shared<TabletSchema>(*tablet_schema);
+    }
+
+    void set_tablet_schema(const TabletSchema& tablet_schema) {
+        TabletSchemaPB* ts_pb = _rowset_meta_pb.mutable_tablet_schema();
+        tablet_schema.to_schema_pb(ts_pb);
+        CHECK(_schema == nullptr);
+        _schema = std::make_shared<TabletSchema>(tablet_schema);
+    }
```

Comment:
Two set_tablet_schema methods here is duplicated? Does either one is enough?

Reply:
fixed